hs/app/reach/Main.hs

```diff
@@ -2450,6 +2451,149 @@ whoami = command "whoami" $ info f fullDesc
   where
     f = pure . script $ write [N.text| echo "$whoami'" |]
 
+type DeviceCode = Text
+
+-- https://stackoverflow.com/a/60790430
+newtype SomeValue = SomeValue String
+instance FromJSON SomeValue where
+  parseJSON = withObject "SomeValue" $ \o -> SomeValue <$> o .: "html_url"
+
+support :: Subcommand
+support = command "support" $ info (pure step1) d
+  where
+    d = progDesc "Upload index.rsh and index.mjs to help us troubleshoot!"
+
```

Comment:
Yuck

Reply:
Formatting?

Reply:
I think [64f88fa](https://github.com/reach-sh/reach-lang/pull/762/commits/64f88fa91beb76eb9c4f410817698a7d1bbdb424) addresses this.